Cargo.toml

```diff
@@ -48,3 +48,6 @@ verifier = []
 
 [patch.'https://github.com/Sovereign-Labs/sovereign.git']
 sovereign-sdk = { path = "../sovereign/sdk" }
+
+[patch.'https://github.com/Sovereign-Labs/nmt-rs.git']
```

Comment:
This is going to be removed afrer https://github.com/Sovereign-Labs/nmt-rs/pull/2 is merged